css-layout-classes/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title> CSS Layout Classes </title>
+</head>
+<body>
+
+  <div class="container">
+
+    <div class="row">
+      <div class="column-full">
+        <h1>Choose your character!</h1>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-one-third">
+        <img src="images/link.png" alt="Link">
+      </div>
+      <div class="column-one-third">
+        <h2>Super Smash Bros. Ultimate</h2>
+        <p>Super Smash Bros. Ultimate received universal acclaim from both critics and players, with some critics calling it the best installment in the series. It received praise for its large amount of content and fine-tuning of existing Smash gamplay elements, although its online mode was widely criticized.</p>
+      </div>
+      <div class="column-one-third">
+        <img src="images/yoshi.png" alt="Yoshi">
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-one-fourth">
+        <img src="images/kirby.png" alt="Kirby">
+      </div>
+      <div class="column-one-fourth">
+        <img src="images/fox.png" alt="Fox">
+      </div>
+      <div class="column-one-fourth">
+        <img src="images/pikachu.png" alt="Pikachu">
+      </div>
+      <div class="column-one-fourth">
+        <img src="images/ness.png" alt="Ness">
+      </div>
+
+      <div class="row">
+        <div class="column-two-thirds">
+          <h2>Donkey Kong</h2>
+          <p>Donkey Kong is a series of video games featuring the adventures of a gorilla called Donkey Kong, conceived by Shigeru Miyamoto in 1981. The franchise consists mainly of two game genres, but also includes additional spin-off titles of various genres.
+          </p>
+          <p>The games of the first genre are mostly single-screen platform/action puzzle types, featuring Donkey Kong as the opponent in an industrial construction setting.</p>
+        </div>
+        <div class="column-one-third">
+          <img src="images/dk.png" alt="Donkey Kong">
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/mario.png" alt="Mario">
+        </div>
+        <div class="column-two-thirds">
+          <h2>Mario</h2>
+          <p>Mario is a fictional character in the Mario video game franchise, owned by Nintendo and created by Japanese video game designer Shigeru Miyamoto. Acting as the company's mascot, as well as being the eponymous protagonist of the series, Mario has appeared in over 200 video games since his creation.</p>
+          <p>Depicted as a short, pudgy, Italian plumber who resides in the Mushroom Kingdom, his adventures generally center upon rescuing Princess Peach from the Koopa villain Bowser.</p>
+        </div>
+      </div>
```

Comment:
You have these two rows nested inside of the row with the four images, they should not be inside that row.